src/main.zig

```diff
@@ -1703,6 +1703,16 @@ fn buildOutputType(
                     }
                     emit_implib = .{ .yes = linker_args.items[i] };
                     emit_implib_arg_provided = true;
+                } else if (mem.eql(u8, arg, "-undefined")) {
```

Comment:
Quick question- you mentioned that this supports `-Wl,"-undefined=dynamic_lookup"`, but it looks like this actually implements `-Wl,-undefined,dynamic_lookup`. Is this intentional?

Reply:
Uhm, wait, did I make a typo somewhere? It should implement the former, `-Wl,"-undefined=dynamic_lookup"` unless I messed something up.

Reply:
Hmm, I think that's what I did. Note that there is another `if` statement after we match `-undefined`:

```zig
    5                 } else if (mem.eql(u8, arg, "-undefined")) {
     4                     i += 1;
     3                     if (i >= linker_args.items.len) {
     2                         fatal("expected linker arg after '{s}'", .{arg});
     1                     }
  1711                     if (mem.eql(u8, "dynamic_lookup", linker_args.items[i])) {
     1                         linker_allow_shlib_undefined = true;
     2                     } else {
     3                         fatal("unsupported -undefined option '{s}'", .{linker_args.items[i]});
     4                     }
     5                 } else {
```

Reply:
Never mind- I forgot that we had this code up there:

```zig
                            // Handle nested-joined args like `-Wl,-rpath=foo`.
                            // Must be prefixed with 1 or 2 dashes.
                            if (linker_arg.len >= 3 and linker_arg[0] == '-' and linker_arg[2] != '-') {
                                if (mem.indexOfScalar(u8, linker_arg, '=')) |equals_pos| {
                                    try linker_args.append(linker_arg[0..equals_pos]);
                                    try linker_args.append(linker_arg[equals_pos + 1 ..]);
                                    continue;
                                }
                            }
```

all good :+1: